proposals/wgs/training/wg-charter-training.md

```diff
@@ -0,0 +1,36 @@
+# WG Training Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Training covers developing, deploying, and operating training jobs on Kubeflow.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- APIs used for running distributed training jobs (e.g., TFJob API)
+- Tools and documentation to aid in ecosystem tool interoperability around distributed training jobs (e.g., TFJob CRD/Controller)
+
+#### Cross-cutting and Externally Facing Processes
+
+- Coordinating with WG Pipeline to make sure distributed training jobs can interact well with pipelines
```

Comment:
@gaocegege What do you think about adding: "coordination with WG AutoML" here also?
One of the main goals is to be able to use all Kubeflow training operators in Katib.
/cc @johnugeorge

Reply:
One concern I have is that there maybe communication overhead between WGs, especially with WG AutoML. Since the co-leads of WG AutoML are almost identical to co-leads of WG Training, would it be possible to merge the efforts? Others might argue that both HPO and NAS belong to training. 

I am okay with separating the two groups though but just thought it would be good to bring this discussion here and avoid confusion at an early stage.

Reply:
Personally, I am glad to merge them. What I worry about is if we split the WG in the future. Maybe we will have more than one project about AutoML (e.g. another project for NAS), the scope of the merged WG may be a little huge.

WDYT @johnugeorge

Reply:
Actually, I think AutoML Kubeflow project can grow up in the future and it would be great if we can involve more maintainers for it.
In our [ROADMAP](https://github.com/kubeflow/katib/blob/master/ROADMAP.md#other-features) we have goals to support AutoMC and Auto Feature Engineering, so it can be also sub-projects.
Also, Katib includes various components, like UI, DB-manager, Controller, which can be maintained by different people later.

As @gaocegege mentioned, I believe it can be hard to manage all of these in Training WG, because it is very differ from the Kubeflow Training controllers design.

Reply:
Yeah, one of the main problems of katib now is the lack of contributors/maintainers. Maybe establishing a separate WG is helpful to enlarge the audience.

Reply:
@andreyvelich Can you help establish WG AutoML if we decide to have it? I think you are familiar with the scope of our AutoML systems than me and Johnu.

Reply:
@gaocegege Sure.

Reply:
Okay I have no objections so it's up to you to decide.

Reply:
@andreyvelich Please help us create WG AutoML, thanks.

Reply:
Also vote for separate WG. Based on my understanding, AutoML is supposed to be a consumer of WG Training. The  concentration is a little bit different.

Reply:
I think we come to an agreement. Ref #358 

I am resolving it.